.travis/klee.sh

```diff
@@ -204,6 +204,17 @@ else
   lit -v test/
 fi
 
+# If metaSMT is the only solver, then rerun lit tests with non-default metaSMT backends
+if [ "X${SOLVERS}" == "XmetaSMT" ]; then
+  metasmt_default=`echo "${METASMT_DEFAULT,,}"` # klee_opts and kleaver_opts use lowercase
+  available_metasmt_backends = "btor stp z3"
+  for backend in $metasmt_default; do
```

Comment:
You're looping over the wrong variable there. It should be ${available_metasmt_backends} not `${metasmt_default}`